plugins/lua/buildingplan/planneroverlay.lua

```diff
@@ -307,10 +318,10 @@ end
 
 PlannerOverlay = defclass(PlannerOverlay, overlay.OverlayWidget)
 PlannerOverlay.ATTRS{
-    default_pos={x=5,y=9},
+    default_pos={x=5,y=8},
```

Comment:
I don't really want to change `overlay` significantly to implement versioning right before the stable release. We should avoid changing the `default_pos` value until that's implemented.

Reply:
I think the only change necessary would be to add a full-width Panel around the MINI_BUTT Label that clears the background.

Reply:
this makes me sad, but okay. 
i mean, it feels kind of putting a band-aid on it 😕 this will need to be addressed later